lms/djangoapps/instructor_analytics/tests/test_basic.py

```diff
@@ -106,17 +106,23 @@ def test_enrolled_students_features_username(self):
             self.assertIn(userreport['username'], [user.username for user in self.users])
 
     def test_enrolled_students_features_keys(self):
-        query_features = ('username', 'name', 'email')
+        query_features = ('username', 'name', 'email', 'city', 'country',)
+        for user in self.users:
```

Comment:
One more request-- please include a user without city and country so that we can verify the code works when those are not specified (assert the expected values).